nextest-runner/src/target_runner.rs

```diff
@@ -7,6 +7,13 @@ use crate::errors::TargetRunnerError;
 use camino::Utf8PathBuf;
 use target_spec::Platform;
 
+#[derive(serde::Deserialize, Debug)]
+#[serde(untagged)]
```

Comment:
Could you add an `expecting` here?

Reply:
Not sure what you mean.

Reply:
Oh, I thought serde macros took an "expecting" atrribute. Doesn't look like they do. Never mind!